client/store.js

```diff
@@ -55,6 +55,16 @@ store.getCurrentAccountId = function() {
     return data.currentAccountId;
 }
 
+store.getDefaultAccountId = function() {
+    return data.settings.get('defaultAccountId');
+}
+
+store.setDefaultAccountId = function(id) {
+    //Do not use store.changeSetting so that the change is immediately visible
```

Comment:
OK, but you'd have to emit the State.settings just after the call to `data.settings.set` (not dispatching an event, but calling `Events.emit(State.settings)` directly, as it's synchronous).